scipy 1.17.0 FTBFS in venvs

```
  [scipy-1.17.0]   [spkg-install] FAILED: [code=1] scipy/linalg/_fblasmodule.c scipy/linalg/_fblas-f2pywrappers.f 
  [scipy-1.17.0]   [spkg-install] /opt/homebrew/opt/python@3.14/bin/python3.14 /Users/runner/work/passagemath/passagemath/.tox/local-homebrew-macos-opthomebrew-standard/local/var/tmp/sage/build/scipy-1.17.0/src/tools/generate_f2pymod.py ../scipy/linalg/fblas.pyf.src -o scipy/linalg --free-threading
  [scipy-1.17.0]   [spkg-install] Traceback (most recent call last):
  [scipy-1.17.0]   [spkg-install]   File "/Users/runner/work/passagemath/passagemath/.tox/local-homebrew-macos-opthomebrew-standard/local/var/tmp/sage/build/scipy-1.17.0/src/tools/generate_f2pymod.py", line 311, in <module>
  [scipy-1.17.0]   [spkg-install]     main()
  [scipy-1.17.0]   [spkg-install]     ~~~~^^
  [scipy-1.17.0]   [spkg-install]   File "/Users/runner/work/passagemath/passagemath/.tox/local-homebrew-macos-opthomebrew-standard/local/var/tmp/sage/build/scipy-1.17.0/src/tools/generate_f2pymod.py", line 305, in main
  [scipy-1.17.0]   [spkg-install]     raise RuntimeError(f"Processing {fname_pyf} with f2py failed!\n"
  [scipy-1.17.0]   [spkg-install]                        f"{out.decode()}\n"
  [scipy-1.17.0]   [spkg-install]                        f"{err.decode()}")
  [scipy-1.17.0]   [spkg-install] RuntimeError: Processing scipy/linalg/fblas.pyf with f2py failed!
  [scipy-1.17.0]   [spkg-install] 
  [scipy-1.17.0]   [spkg-install] /opt/homebrew/opt/python@3.14/bin/python3.14: Error while finding module specification for 'numpy.f2py' (ModuleNotFoundError: No module named 'numpy')
  [scipy-1.17.0]   [spkg-install] 
  [scipy-1.17.0]   [spkg-install] [2/1254] Copying file scipy/optimize/cython_optimize/__init__.py
  [scipy-1.17.0]   [spkg-install] [3/1254] Generating from 'src/dfitpack.pyf'
  [scipy-1.17.0]   [spkg-install] FAILED: [code=1] scipy/interpolate/_dfitpack.cpython-314-darwin.so.p/_dfitpackmodule.c scipy/interpolate/_dfitpack.cpython-314-darwin.so.p/_dfitpack-f2pywrappers.f 
  [scipy-1.17.0]   [spkg-install] /opt/homebrew/opt/python@3.14/bin/python3.14 /Users/runner/work/passagemath/passagemath/.tox/local-homebrew-macos-opthomebrew-standard/local/var/tmp/sage/build/scipy-1.17.0/src/tools/generate_f2pymod.py ../scipy/interpolate/src/dfitpack.pyf -o scipy/interpolate/_dfitpack.cpython-314-darwin.so.p '--f2cmap=""' --free-threading
  [scipy-1.17.0]   [spkg-install] Traceback (most recent call last):
  [scipy-1.17.0]   [spkg-install]   File "/Users/runner/work/passagemath/passagemath/.tox/local-homebrew-macos-opthomebrew-standard/local/var/tmp/sage/build/scipy-1.17.0/src/tools/generate_f2pymod.py", line 311, in <module>
  [scipy-1.17.0]   [spkg-install]     main()
  [scipy-1.17.0]   [spkg-install]     ~~~~^^
  [scipy-1.17.0]   [spkg-install]   File "/Users/runner/work/passagemath/passagemath/.tox/local-homebrew-macos-opthomebrew-standard/local/var/tmp/sage/build/scipy-1.17.0/src/tools/generate_f2pymod.py", line 305, in main
  [scipy-1.17.0]   [spkg-install]     raise RuntimeError(f"Processing {fname_pyf} with f2py failed!\n"
  [scipy-1.17.0]   [spkg-install]                        f"{out.decode()}\n"
  [scipy-1.17.0]   [spkg-install]                        f"{err.decode()}")
  [scipy-1.17.0]   [spkg-install] RuntimeError: Processing ../scipy/interpolate/src/dfitpack.pyf with f2py failed!
  [scipy-1.17.0]   [spkg-install] 
  [scipy-1.17.0]   [spkg-install] /opt/homebrew/opt/python@3.14/bin/python3.14: Error while finding module specification for 'numpy.f2py' (ModuleNotFoundError: No module named 'numpy')
  [scipy-1.17.0]   [spkg-install] 
  [scipy-1.17.0]   [spkg-install] [4/1254] Scanning target fitpack_lib for modules
  [scipy-1.17.0]   [spkg-install] [5/1254] Generating scipy/_cyutility.c with a custom command
  [scipy-1.17.0]   [spkg-install] ninja: build stopped: subcommand failed.
```
https://github.com/passagemath/passagemath/actions/runs/21053570053/job/60549485467?pr=1982#step:10:3293

Also seen on ubuntu-plucky-standard https://github.com/passagemath/passagemath/actions/runs/21129053756/job/60755913184#step:11:8847

--- a/build/pkgs/scipy/patches/0001-tools-generate_f2pymod.py-Do-not-try-to-invoke-f2py-.patch
+++ b/build/pkgs/scipy/patches/0001-tools-generate_f2pymod.py-Do-not-try-to-invoke-f2py-.patch
@@ -1,0 +1,27 @@
+From c6361487f5fe5609dfec16d18ecbd381c2b8c1d5 Mon Sep 17 00:00:00 2001
+From: Matthias Koeppe <mkoeppe@math.ucdavis.edu>
+Date: Mon, 19 Jan 2026 17:04:07 -0800
+Subject: [PATCH] tools/generate_f2pymod.py: Do not try to invoke f2py via
+ sys.executable
+
+---
+ tools/generate_f2pymod.py | 3 +--
+ 1 file changed, 1 insertion(+), 2 deletions(-)
+
+diff --git a/tools/generate_f2pymod.py b/tools/generate_f2pymod.py
+index aaedad2bd2..b5cec7dff7 100644
+--- a/tools/generate_f2pymod.py
++++ b/tools/generate_f2pymod.py
+@@ -293,8 +293,7 @@ def main():
+ 
+     # Now invoke f2py to generate the C API module file
+     if args.infile.endswith(('.pyf.src', '.pyf')):
+-        cmd = [sys.executable, '-m', 'numpy.f2py', fname_pyf,
+-               '--build-dir', outdir_abs] + nogil_arg
++        cmd = ['f2py', fname_pyf, '--build-dir', outdir_abs] + nogil_arg
+         if args.f2cmap:
+             cmd += ['--f2cmap', args.f2cmap]
+ 
+-- 
+2.51.1
+

